api/researches/urls.py

```diff
@@ -25,4 +25,5 @@
     path('get-direction-params', views.get_direction_params),
     path('localization', views.localization),
     path('localization/save', views.localization_save),
+    path('field-stattalon', views.required_fields_stattalon),
```

Comment:
Давай всё понятнее сделаем
```suggestion
    path('required-stattalon-fields', views.required_stattalon_fields),
```